### DISCUSSION TITLE
ASA Mod - In Game Shop - Add method to ensure good rewards

### DISCUSSION THREAD

**Discussion by Astaroth368** [gameserverapp/Platform#618] [Suggestions]
Using `GiveItemToPlayer` with quality `x` spawns such varied items. Players spend money on tokens to use in the shop and could end up with poor rewards. Is it possible for the GSA mod to add a new script command or something to spawn an item with more specific stats (E.g.  specify min and max armour and durability)?

Thanks

**Comment by mvaessen** [comment]
When the tribute uploading comes back we might be able to do such things with GSA again, but until then is my advice: best to look for mods that have this feature baked in. No plans to add this to GSA mod.
